templates/security/cis.html

```diff
@@ -99,7 +100,7 @@ <h2>Which versions of Ubuntu have CIS tooling?</h2>
       </tbody>
     </table>
     <p>
-      <a href="/security/certifications#get-in-touch" class="p-button--positive js-invoke-modal">Access the Ubuntu CIS benchmark tooling</a>
+      <a href="/advantage" class="p-button--positive js-invoke-modal">Access the Ubuntu CIS benchmark tooling</a>
```

Comment:
Can you remove the js-invoke-modal class, as this will trigger the previous contact form. Thanks

Reply:
done